nntrainer/models/neuralnet.cpp

```diff
@@ -1275,6 +1278,12 @@ void NeuralNetwork::exports(const ml::train::ExportMethods &method,
 #endif
     break;
   }
+  case ml::train::ExportMethods::METHOD_FLATBUFFER: {
+
```

Comment:
I think it might be better to mention NYI if it is not yet implemented to reduce the confusion.
something like,
``` c++
case ml::train::ExportMethods::METHOD_FLATBUFFER: {

   \* NYI *\

    model_graph.deallocateTensors();
    model_graph.allocateTensors(ExecutionMode::TRAIN);
    break;
  }

```